src/clr-angular/data/datagrid/datagrid-column-separator.ts

```diff
@@ -59,12 +59,11 @@ export class ClrDatagridColumnSeparator {
     this.renderer.setStyle(this.document.body, 'cursor', 'auto');
   }
 
+  private withinMaxResizeRange: boolean;
   private redFlagTracker(resizeTrackerEl: HTMLElement) {
-    let isWithinMaxResizeRange: boolean;
-    // @TODO(JEREMY) Review this, it will always be true because above is always null
-    if (isWithinMaxResizeRange !== this.columnResizerService.isWithinMaxResizeRange) {
-      isWithinMaxResizeRange = this.columnResizerService.isWithinMaxResizeRange;
-      if (!isWithinMaxResizeRange) {
+    if (this.withinMaxResizeRange !== this.columnResizerService.isWithinMaxResizeRange) {
+      this.withinMaxResizeRange = this.columnResizerService.isWithinMaxResizeRange;
```

Comment:
Q: Could we eliminate the private `withinMaxResizeRange` and just tuse the value on the resize service directly?

Reply:
We shouldn't eliminate that property. If we eliminate that, we will try to set the same class to the DOM element on every drag, and drag event fires a lot. So it's there to prevent adding the same class if the DOM element has that class already.